### DISCUSSION TITLE
Can't use the classes

### DISCUSSION THREAD

**Discussion by Carlitooo2** [tailwindlabs/tailwindcss#12019] [Help] [answered]
I have followed the steps in the documentation, tried the chatgpt. Now I'm at the dead end, I cant use the classes in my index.html despite having tailwindcss installed and linked to the head of my html.

Can somebody help me how to fix this? Thank you in advance

**Comment by wongjn** [comment]
Consider checking:
- Tailwind has compiled since you added any classes.
- The `content` file globs sufficiently cover the `index.html` file.
- [String concatenation/interpolation is not used to build class names](https://tailwindcss.com/docs/content-configuration#dynamic-class-names). 
- The class names would resolve to Tailwind classes that Tailwind would generate rules for.

**Comment by Carlitooo2** [comment, marked as answer]
It seems that I have found my answer. I didnt add the play CDN Script which is the `<script src="https://cdn.tailwindcss.com"></script>` in my index.html. Sorry for the trouble, thanks!
